docs/user-guide/rules/list.md

```diff
@@ -120,6 +120,7 @@ Within each cateogory, the rules are grouped by the [_thing_](http://apps.workfl
 
 - [`function-allowed-list`](../../../lib/rules/function-allowed-list/README.md): Specify a list of allowed functions.
 - [`function-disallowed-list`](../../../lib/rules/function-disallowed-list/README.md): Specify a list of disallowed functions.
+- [`function-no-unknown`](../../../lib/rules/function-no-unknown/README.md): Disallow unknown functions.
```

Comment:
Should the rule be in the "Avoid errors" section?

Reply:
That makes sense! Done.